bin/travis

```diff
@@ -3,9 +3,9 @@
 set -e
 
 if [[ $TRAVIS_SCALA_VERSION == 2.11* ]] && [[ $SCALAZ_VERSION = 7.2* ]]; then
-  SBT_COMMAND=";coverage ;clean ;test ;makeSite ;coverageOff"
+  SBT_COMMAND="\"set scalazVersion in ThisBuild := \\\"7.2.1\\\"\" ;coverage ;clean ;test ;makeSite ;coverageOff"
```

Comment:
I think it needs a leading semi-colon and less quoting.  This works for me in a shell:

```sh
SBT_COMMAND=";set scalazVersion in ThisBuild := \"7.2.1\" ;test ;mimaReportBinaryIssues"
```